src/Utilities/Compiler/Options/AnalyzerOptionsExtensions.cs

```diff
@@ -163,13 +163,33 @@ static bool TryParseValue(string value, out ImmutableHashSet<TEnum> result)
             CancellationToken cancellationToken)
             => options.GetSymbolNamesOption(EditorConfigOptionNames.DisallowedSymbolNames, namePrefixOpt: null, rule, compilation, cancellationToken);
 
+
+        public static SymbolNamesOption GetAdditionalRequiredSuffixesOption(
+            this AnalyzerOptions options,
+            DiagnosticDescriptor rule,
+            Compilation compilation,
+            CancellationToken cancellationToken)
+        {
+            return options.GetSymbolNamesOption(EditorConfigOptionNames.AdditionalRequiredSuffixes, namePrefixOpt: "T:", rule, compilation, cancellationToken, GetParts);
+
+            static SymbolNamesOption.NameParts GetParts(string name)
+            {
+                var lastIndexOfColon = name.LastIndexOf(':');
```

Comment:
Looking at the examples under "Methods" section in https://github.com/dotnet/csharplang/blob/master/spec/documentation-comments.md#id-string-format, it seems `:` can be present inside the doc comment ID for few symbol kinds. Can we choose a different separator character? Maybe `::` or `->`?

Reply:
You can probably also check that there is a single separator in the value, instead of using LastIndex, for a valid suffix.

Reply:
I think it would also be good to invoke Trim() on each of the name parts, probably in the constructor?